infrastructure/environments/dev/provider.tf

```diff
@@ -0,0 +1,21 @@
+terraform {
+  backend "azurerm" {
+    container_name       = "terraformstate-appeals-back-office"
+    resource_group_name  = "pins-rg-shared-terraform-uks"
+    subscription_id      = "edb1ff78-90da-4901-a497-7e79f966f8e2"
+    storage_account_name = "pinssttfstateuksappealsbackoffice"
+    # key                  = "dev.tfstate"
+  }
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 3.99.0"
+    }
+  }
+  required_version = ">= 1.1.6, < 3.99.0"
```

Comment:
If I follow the docs correctly, this should be the terraform version, currently we use 1.7.3.

For DaRT I've gone for: `>= 1.7.3, < 1.8`